### ISSUE TITLE
Return status code 404 when Page Not Found

### ISSUE THREAD

**Issue by kyleoliveiro** [interactivethings/catalog#429]
Currently, a status code of 200 is returned regardless of whether the requested page exists or not. Ideally, the status code should be 404.

**Comment by herrstucki** [comment]
Can you elaborate on the scenario where this doesn't work? Maybe provide a minimal reproducible setup?

Note that this probably only is the case during development (i.e. when using `catalog start`). Once built, Catalog isn't responsible for serving your pages.

**Comment by kyleoliveiro** [comment]
Yes, it's during development. I'm using Cypress to check for pages which should exist in the documentation by visiting a URL and asserting that the response code is 200.

**Comment by wereHamster** [comment]
This project has been deprecated and is no longer maintained.
